CMakeLists.txt

```diff
@@ -66,8 +68,8 @@ catkin_package(
     linear_feedback_controller_msgs
     tf
     tf_conversions
-    DEPENDS Franka pinocchio crocoddyl
-  # DEPENDS Franka pinocchio tsid crocoddyl
+    # DEPENDS Franka pinocchio crocoddyl
```

Comment:
```suggestion
```
This commented line shouldn't be there. Simply remove it since there is no need to keep it